examples/basic/cli/src/OntologyType.ts

```diff
@@ -14,10 +14,15 @@
  * limitations under the License.
  */
 
+import type { OsdkObjectFrom } from "@osdk/api";
 import type { Ontology } from "@osdk/examples.basic.sdk";
 
 // This little trick makes it easier for me to see
 // what is happening in the types when I hover.
 // We need to actually generate some nice types here too
 type CapturedOntologyType = typeof Ontology;
 export interface OntologyType extends CapturedOntologyType {}
+
+// Demo for if you want concrete types
+export interface Employee extends OsdkObjectFrom<"Employee", OntologyType> {}
+export interface Todo extends OsdkObjectFrom<"Todo", OntologyType> {}
```

Comment:
Cassie mentioned wanting to do something like this

Reply:
Do you not intend to generate the interfaces like we're doing for v1.1? (will these generated types be usable or just a mess of inference like the v1.1 internal types are?)